x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/moving_attacks_callout/index.tsx

```diff
@@ -5,48 +5,77 @@
  * 2.0.
  */
 import React from 'react';
-import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+import { css } from '@emotion/react';
+import {
+  EuiButton,
+  EuiCallOut,
+  EuiSpacer,
+  useEuiTheme,
+  useIsWithinMaxBreakpoint,
+} from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { SecurityPageName } from '@kbn/deeplinks-security';
 import { SecuritySolutionLinkButton } from '../../../common/components/links';
 import { useMovingAttacksCallout } from './use_moving_attacks_callout';
 import * as i18n from './translations';
 
 export const CALLOUT_TEST_DATA_ID = 'moving-attacks-callout' as string;
+export const HIDE_BUTTON_TEST_DATA_ID = 'hide-callout-button' as string;
 
 /**
  * Component to display a moving attacks callout
  */
 export const MovingAttacksCallout: React.FC = React.memo(() => {
+  const { euiTheme } = useEuiTheme();
+
   const { isMovingAttacksCalloutVisible, hideMovingAttacksCallout } = useMovingAttacksCallout();
 
+  const isSmall = useIsWithinMaxBreakpoint('m');
+  const calloutDescriptionWidth = css`
+    max-width: ${isSmall ? 'auto' : '1000px'};
+  `;
+
+  const hideButtonSpacing = css`
+    margin-left: ${euiTheme.size.s};
+  `;
+
   return isMovingAttacksCalloutVisible ? (
     <>
       <EuiCallOut
         announceOnMount
-        onDismiss={hideMovingAttacksCallout}
         title={
           <FormattedMessage
             id="xpack.securitySolution.attackDiscovery.movingAttacksCallout.title"
-            defaultMessage="Attack discovery is moving to Detections"
+            defaultMessage="Attack Discovery is moving to Detections"
           />
         }
         iconType="bolt"
         data-test-subj={CALLOUT_TEST_DATA_ID}
       >
-        <p>
+        <p css={calloutDescriptionWidth}>
           <FormattedMessage
             id="xpack.securitySolution.attackDiscovery.movingAttacksCallout.description"
-            defaultMessage="You can now schedule scans and view attacks from Detections -> Attacks. Manual scans are still available only on the legacy page. Attack discovery will move completely to the Detections section in a future release."
+            defaultMessage="You can now schedule scans and view promoted attacks from Detections -> Attacks. Manual scans are still available only here, on the legacy page. Attack Discovery will move completely to the Detections section in a future release."
```

Comment:
The page title and navigation are both `Attack discovery`; should it be consistent here?

Reply:
@natasha-moore-elastic @benironside would appreciate your input here.

Just to clarify, in UI we use `Attack discovery` and in the callout we spelled as `Attack Discovery`. We also use the latter one in documentation. Should we unify this?